.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,14 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/anaconda
+{
+	"name": "Anaconda (Python 3)",
+	"build": { 
+		"context": "..",
+		"dockerfile": "Dockerfile"
```

Comment:
The only comment i have is that if you might nod need the `Dockerfile` file as long as you just define everything you need in the `features` property. There should be a python3 feature https://github.com/devcontainers/features/tree/main/src/python. This is really the tiniest optimisation and should not block this PR, maybe we can just open an issue.

Reply:
Thanks Max! I do have to say that VS Code devcontainers are a bit unfamiliar.  I have created an issue [here](https://github.com/cds-snc/sentinel-forward-data-action/issues/3).